src/Panel/User.php

```diff
@@ -73,19 +73,40 @@ public function dropdown(array $options = []): array
 			'disabled' => $this->isDisabledDropdownOption('changeRole', $options, $permissions)
 		];
 
+		$result[] = [
+			'dialog'   => $url . '/changeLanguage',
+			'icon'     => 'translate',
+			'text'     => I18n::translate('user.changeLanguage'),
+			'disabled' => $this->isDisabledDropdownOption('changeLanguage', $options, $permissions)
+		];
+
+		$result[] = '-';
+
 		$result[] = [
 			'dialog'   => $url . '/changePassword',
 			'icon'     => 'key',
 			'text'     => I18n::translate('user.changePassword'),
 			'disabled' => $this->isDisabledDropdownOption('changePassword', $options, $permissions)
 		];
 
-		$result[] = [
-			'dialog'   => $url . '/changeLanguage',
-			'icon'     => 'translate',
-			'text'     => I18n::translate('user.changeLanguage'),
-			'disabled' => $this->isDisabledDropdownOption('changeLanguage', $options, $permissions)
-		];
+		if ($this->model->kirby()->system()->is2FAWithTOTP() === true) {
+			if ($account || $this->model->kirby()->user()->isAdmin()) {
```

Comment:
I think we can combine these two `if`s into one.

Reply:
Tbh, I think that just harms legibility in this case. I think it's good to read them as two distinct steps/checks.

Reply:
I think the following is just as readable but cleaner:

```php
if (
    $this->model->kirby()->system()->is2FAWithTOTP() === true &&
    ($account || $this->model->kirby()->user()->isAdmin())
) {
```

If you disagree, I'm fine with keeping it as it is.